templates/vaults/crypt-vault.tmpl.yaml

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
can you elaborate on having 2 different vaults? especially if vaults appear to be going away with: https://github.com/advancedtelematic/ota-tuf/pull/166

Reply:
I'm also not a fan of having multiple vault deployments. We will be removing this as soon after the linked PR has been merged.